codec_test.go

```diff
@@ -49,3 +49,47 @@ func TestCodec(t *testing.T) {
 	require.Equal(t, math.MaxInt32, targetInt, "expected codec to successfully decode int")
 	require.Equal(t, 0, len(buf.Bytes()), "expected buffer to be empty")
 }
+
+func TestMap(t *testing.T) {
+	var buf bytes.Buffer
+	mh := &codec.MsgpackHandle{WriteExt: true}
+	enc := codec.NewEncoder(&buf, mh)
+	dec := codec.NewDecoder(&buf, mh)
+
+	var m map[string]string = map[string]string{
+		"hello": "world",
+		"foo":   "bar",
+	}
+	err := enc.Encode(m)
+	require.Nil(t, err, "expected encoding to succeed")
+
+	var targetMap map[string]string
+	err = dec.Decode(&targetMap)
+	require.Nil(t, err, "expected decoding to succeed")
+	require.Equal(t, m, targetMap)
+	require.Equal(t, "world", targetMap["hello"])
```

Comment:
No need for these two lines, right? Since the first `require.Equal` already does a DeepEqual